src/main/java/com/meplato/store2/products/Product.java

```diff
@@ -13,7 +13,12 @@
  */
 package com.meplato.store2.products;
 
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
```

Comment:
Was all this really missing? ouch

Reply:
Oh, forgot to delete them in that file actually. The thing with the `apigen` is - it's kind of semi-automatic. In case of java - it adds these "default imports" to multiple files, no matter whether they are needed or not. I prefer to manually delete them, after I'm done with the actual changes, to keep the code cleaner, rather than leaving them in to save us the manual effort each time. It's not as trivial to generate java files with all the correct imports they need afaik.

Reply:
Ah, we actually added a formatter https://github.com/meplato/apigen/pull/7